webservices/resources/sched_a.py

```diff
@@ -86,8 +86,7 @@ def amount_column(self):
         'contributor_employer',
         'contributor_occupation',
     ]
-    use_pk_for_count=True
-
+    use_pk_for_count = True
```

Comment:
Thanks for fixing this! I keep fixing it in various WIP PR's that aren't ready for merge 😅